src/Calculator.cpp

```diff
@@ -6,6 +7,161 @@
  */
 ////////////////////////////////////////////////////////////////////////////
 
+#include <algorithm>
+
 #include "Calculator.h"
 
-// Add Function definitions here.
\ No newline at end of file
+// Add Function definitions here.
+Calculator::Calculator() {}
```

Comment:
Why do we need this constructors? if so we can use the default keyboard and have the compiler generate it from you